### DISCUSSION TITLE
Moonlight like a second monitor and not only stream

### DISCUSSION THREAD

**Discussion by nano02g** [LizardByte/community#413] [Sunshine]
Hi , i need some help , is ther a manner tu use moonlight like a second monitor and not only to stream the principal monitor ? a thing that you can configurate with 🪟+P , if there is please help me!

**Comment by ganzvu** [comment]
yes, you can use iddsampledriver (theres also a hdr fork for window 11) to utilize sunshine & moonlight as a second monitor combo.

**Reply by nano02g** [reply]
How can I do that, can you help me please?

**Reply by ganzvu** [reply]
please refer to these Reddit thread as they helped me

Tutorial on installing the driver as a second monitor
https://www.reddit.com/r/MoonlightStreaming/comments/rzpcpc/moonlight_streaming_without_monitor_no_dummy_plug/

HDR setup 
https://www.reddit.com/r/cloudygamer/comments/185agmk/guide_how_to_setup_hdr_with_moonlightsunshine/

[a comment by an automated account is not shown]
